<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search_window</class>
 <widget class="QMainWindow" name="search_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="search_box_warning">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_box">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>331</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="search_result">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>541</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_file_button">
    <property name="geometry">
     <rect>
      <x>524</x>
      <y>50</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse Folder</string>
    </property>
   </widget>
   <widget class="QTreeView" name="search_tree">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>971</width>
      <height>381</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="animated">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
